assets/models/system/compvis-stable-diffusion-v1-4/description.md

```diff
@@ -32,7 +125,7 @@ Batch |<a href="https://aka.ms/azureml-infer-batch-sdk-safe-text-to-image" targe
 }
 ```
 
-#### Sample output
+## Sample output
 
 ```json
 [
```

Comment:
should this be: 
`"nsfw_content_detected": false` instead of `"nsfw_content_detected": False`

Reply:
good catch